### ISSUE TITLE
"Create/update local copies" doesn't work

### ISSUE THREAD

**Issue by DarkShadow44** [rimpy-custom/RimPy#81]
I'm on Arch Linux, I tried to use the local mod functionality.
I went on the "Local Copies" tab, selected the "Harmony" mod and clicked "Create/update local copies", but nothing happens.

This is the log that got printed to the console:
```(Mod: Harmony
Steam: True
Author: Andreas Pardeike
Game version: 1.2, 1.3
Mod version: 1.4.0.0
Path: /run/media/fabian/DataB/_Linux/Steam/steamapps/workshop/content/294100/2009463077
Image: /run/media/fabian/DataB/_Linux/Steam/steamapps/workshop/content/294100/2009463077/About/Preview.png, None)
```

**Comment by rimpy-custom** [comment]
That feature is not implemented yet. It is in DEV build only. I didn't mention that anywhere. Do not close this issue, for information purposes, please.

**Comment by Skullywag** [comment]
This doesnt work even on the latest DEV build FYI (clicking seem to do nothing, better than crashing as it was before), with 1.4 on the way and the new DLC being almost released, I imagine you will get a few more people asking for this :D

**Comment by bjornnorberg** [comment]
Not working for me either 1.2.6.25 windows version

**Comment by rimpy-custom** [comment]
Closing for now. Reopen if issue wasn't fixed with latest Linux build 1.2.6.28.
